lib/components/config/ConfigButtonItem.dart

```diff
@@ -21,21 +20,5 @@ class ConfigButtonItem extends StatelessWidget {
           constraints: BoxConstraints(minHeight: Utils.minInteractiveSize, minWidth: double.infinity),
           child: Center(child: content),
         ));
-
```

Comment:
This change seems kind of major compared to the other API fixes in this PR. Can you explain why we are removing this?

Reply:
It's unreachable code I left around for whatever reason